src/esp/bindings/PhysicsObjectBindings.cpp

```diff
@@ -457,12 +457,13 @@ void declareArticulatedObjectWrapper(py::module& m,
             "link_id. Use link_id==-1 to get the base link.")
                .c_str(),
            "link_id"_a)
-      .def("get_link", &ManagedArticulatedObject::getLink,
-           ("Get this " + objType +
-            "'s articulated link specified by the passed "
-            "link_id. Use link_id==-1 to get the base link.")
-               .c_str(),
-           "link_id"_a)
+      // No binding class currently exists for the Articulated Link
+      // .def("get_link", &ManagedArticulatedObject::getLink,
+      //      ("Get this " + objType +
+      //       "'s articulated link specified by the passed "
+      //       "link_id. Use link_id==-1 to get the base link.")
+      //          .c_str(),
+      //      "link_id"_a)
```

Comment:
Unrelated: Is this planned?

Reply:
Yes, @aclegg3 and I have been discussing a mechanism for this.